service/src/test/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreatePostgresqlDbStepTest.java

```diff
@@ -92,7 +100,9 @@ void doStepSuccess() throws InterruptedException {
             LandingZoneFlightMapKeys.BILLING_PROFILE,
             new ProfileModel().id(UUID.randomUUID()),
             LandingZoneFlightMapKeys.LANDING_ZONE_ID,
-            LANDING_ZONE_ID),
+            LANDING_ZONE_ID,
+            LandingZoneTagKeys.PGBOUNCER_ENABLED.toString(),
+            "true"),
```

Comment:
could you add an assert for the `pgbouncer.enabled` tag? See line ~145 where this test calls `verifyBasicTags`, hopefully you can copy/paste from that implementation to check the pgbouncer tag.